### PR TITLE
nuget-package-explorer: Change url to Chocolatey

### PR DESCRIPTION
**Pull request by niheaven**
Add hash from Chocolatey.

Please also consider https://github.com/lukesampson/scoop/pull/3249

### CODE REVIEW

File: bucket/nuget-package-explorer.json [hunk 1]
```diff
@@ -3,15 +3,19 @@
...
     },
     "autoupdate": {
-        "url": "https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/releases/download/v$version/PackageExplorer.$version.zip"
+        "url": "https://packages.chocolatey.org/NugetPackageExplorer.$version.nupkg",
+        "hash": {
+            "url": "https://chocolatey.org/packages/NugetPackageExplorer",
+            "regex": "$sha256.*?$basename"
+        }
     },
-    "bin": "nugetpackageexplorer.exe",
```
Comment (r15ch13) [inline bucket/nuget-package-explorer.json]: Why did you remove the `bin` property?

Comment (niheaven) [reply bucket/nuget-package-explorer.json]: Sorry, I think Nuget Package Explorer is a pure GUI app, but it could open `*.nupkg` through command, so add it again.
